stable/fairwinds-insights/values.yaml

```diff
@@ -41,7 +41,7 @@ openApiImage:
   # -- Docker image repository for the Open API server
   repository: swaggerapi/swagger-ui
   # -- Overrides tag for the Open API server, defaults to image.tag
-  tag: v4.1.3  # NOTE: if you're changing this, be sure to look at templates/openapi-nginx-conf.yaml
+  tag: v5.20.2  # NOTE: if you're changing this, be sure to look at templates/openapi-nginx-conf.yaml
```

Comment:
Did you check this?

Reply:
Yea, that was my topic with Robert today. He said to try this anyways and see if something breaks. He is hoping this will work.